kalite/distributed/templates/distributed/base.html

```diff
@@ -302,6 +291,22 @@
                                 </li>
                               </ul>
                             </li>
+                            <!-- Language selection -->
+                            {% if request.session.facility_user.is_teacher %}
+                            <li class="teacher-only">
+                            {% else %}
+                            <li class="not-admin-only">
+                            {% endif %}
```

Comment:
Shouldn't this be visible for everyone? I think you can replace this conditional with just a `<li>` tag (no class).